db/migrate/20190909183737_ensure_no_exports_corruption.rb

```diff
@@ -0,0 +1,10 @@
+class EnsureNoExportsCorruption < ActiveRecord::Migration[5.2]
+  def self.up
+    remove_column :exports, :bands_string
+    add_column :exports, :bands_string, :text, :null => false
```

Comment:
Won't this irrecoverably delete the `bands_string` values in the production site?

Reply:
@jywarren valid. I didn't realize schema changes would drop data. Thats something I wanted to check on the unstable server! Sorry!!! Thank u for catching that, good learning experience for me.... 

Will revert back to original idea of `before_validation`